addOns/spiderAjax/src/main/resources/org/zaproxy/zap/extension/spiderAjax/resources/Messages.properties

```diff
@@ -1,8 +1,35 @@
-
+# Api Views
+spiderajax.api.view.fullResults = Shows the full spidered content detected by the AjaxSpider.
+spiderajax.api.view.numberOfResults = Shows the number of detected resources.
+spiderajax.api.view.optionBrowserId = Shows the tipe of browser being used for the crawl, this has been set at launch time (The values should be: Chrome, Chrome-headless, Firefox, Firefox-headless, HtmlUnit, PhantomJS, Safari).
+spiderajax.api.view.optionClickDefaultElems = Shows the configured value on clicking actions on defaul html tag elements such as "a" "button" "input", all asociated with some action or links on the page.
+spiderajax.api.view.optionClickElemsOnce = This value is boolean and is for the AjaxSpider to know if it have to click on the links only once.
+spiderajax.api.view.optionEventWait = Shows the event timer in miliseconds.
+spiderajax.api.view.optionMaxCrawlDepth = Shows the configured value for the scan crawl depth.
+spiderajax.api.view.optionMaxCrawlStates = Shows the configured value on how many crawl states it will should show.
+spiderajax.api.view.optionMaxDuration = Shows the configured maxduration of the crawl the vaule is in minutes. 
+spiderajax.api.view.optionNumberOfBrowsers = Shows the configured number of browsers to be used.
+spiderajax.api.view.randominputs = Shows if the AjaxSpider will use random values in form fields when crawling, if set to true.
+spiderajax.api.view.optionReloadWait = Shows the configured time when reload of the mage is made, this value is in miliseconds. Default= 1000.
+spiderajax.api.view.results = Shows the final results of the crawler made to the site.
+spiderajax.pai.view.status = Shows the current status of the crawler. Actual values are Stopped and Running.
+# Api Actions 
+spiderajax.api.action.scan = Runs the spider against the given URL. URL parameter its mandatory and needs to have "http(s)://". inScope -> default its false, if you change this value to true, mean you already have this URL on your context/scope configuration. ContextName -> String value of any of your defined context you have, if the value here is not already defined, it will prompt an error. subtreeOnly -> default value is false, if changed to true it will crawl all the links on the web restricting the spider under the sites resources.
+spiderajax.api.action.scanAsUser = Runs the spider from the perspective of a User of the web aplication. The 'userName' should be previusly defined on the context configuration. We recomend you set the url parameter, otherwise it's used an existing URL from the context (if any).   The parameter 'subtreeOnly' allows to restrict the spider under a site's subtree (using the specified 'url') default value here is false.
+spiderajax.api.action.setOptionBrowserId = Sets the configuration of the AjaxSpider to use one of the following suported browsers: Chrome, Chrome-headless, Firefox, Firefox-headless, HtmlUnit, PhantomJS, Safari.<br/>
```

Comment:
It might be better to also reference the Selenium add-on, the actual values depend on it.